app/controllers/admin/messages/list.js

```diff
@@ -1,36 +1,66 @@
 import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
 
-export default Controller.extend({
-  columns: [
+export default class extends Controller.extend(EmberTableControllerMixin) {
+per_page = 100;
+
+sort_by = 'time';
+
+sort_dir = 'ASC';
+
+@computed()
+get columns() {
+  return [
     {
-      propertyName : 'recipient',
-      title        : 'Recipients'
+      name            : 'Recipients',
+      valuePath       : 'recipient',
+      headerComponent : 'tables/headers/sort',
+      isSortable      : true
     },
     {
-      propertyName : 'action',
-      title        : 'Trigger'
+      name      : 'Trigger',
+      valuePath : 'action'
     },
     {
-      subject  : 'emailSubject',
-      message  : 'emailMessage',
-      title    : 'Email Message',
-      template : 'components/ui-table/cell/cell-title-message'
+      name      : 'Email Message',
+      valuePath : 'emailMessage',
+
+      cellComponent   : 'ui-table/cell/cell-title-message',
+      extraValuePaths : ['emailSubject', 'emailMessage'],
```

Comment:
I saw that you're using extraRecords in template. Then `valuePath` is useless, right?

Reply:
TBH, I just saw the PR made by @CosmicCoder96, As I could not find much about `extraValuePaths` in their docs

Reply:
@kushthedude what exactly was the problem when you were using valuePath for `emailMessage`? Was it not rendering?

Reply:
@kushthedude I'm asking why aren't you using valuePath and repeating it in extraValuePath? Was there any problem in usig valuePath to compare?

Reply:
Blank columns were getting shown.

Reply:
@kushthedude I'm just a bit skeptical about this. Can you just do this once -
- remove `emailMessage` from `extraValuePaths`
- use `record` instead of `extraRecords.emailMessage` in `cell-title-message.hbs`
- Do this similarly for `notificationMessage` below

Please paste a screenshot after these changes here. Btw this is the correct way of doing it so that we don't have repeated vars

Reply:
@shreyanshdwivedi Please see this files 👍 
 https://github.com/CosmicCoder96/open-event-frontend/blob/870c936ea0c42a3afb775b4b8c1deebb6d78291c/app/templates/components/ui-table/cell/admin/users/cell-system-roles.hbs
https://github.com/CosmicCoder96/open-event-frontend/blob/870c936ea0c42a3afb775b4b8c1deebb6d78291c/app/controllers/admin/users/list.js

Since in `Ember Table` if need to access a property of record other than than the value path, We have used `extraValuepath` to specify that property, The following change which you are telling me to do will do nothing but blank columns. 
In your recent PR related to sessions , Since for the column we only needed to use the `name` which we defined in `valuePath` hence `record.name` was feasible but see in above files when we need to use more property of record other than `valuePath`, `extraValuepath` and `extraRecords` are used.
Hope this clears your confusion

Reply:
@kushthedude I think you aren't getting my point. All I'm saying is there must be the use of `record` somewhere in the template but you're only using `extraRecords`
The file you linked here also uses record once. So, please use `valuePath` at least somewhere in the template. I hope you get my point now